stripe/stripe_object.py

```diff
@@ -225,12 +223,31 @@ def refresh_from(
         self._transient_values = self._transient_values - set(values)
 
         for k, v in values.items():
-            super(StripeObject, self).__setitem__(
-                k,
-                util.convert_to_stripe_object(
-                    v, api_key, stripe_version, stripe_account
-                ),
-            )
+            inner_class = self._get_inner_class_type(k)
+            is_dict = self._get_inner_class_is_beneath_dict(k)
+            if is_dict:
+                obj = {
```

Comment:
Are we ok with this "breaking" change of going from StripeObject to dict?

Reply:
Oh, great point. The blast radius is sooo small here (there's only one of these) and in my estimation the chance of a user relying on StripeObjectness and not just using as a dict is miniscule.

Reply:
Added a note to the PR description.

Reply:
(Not going to change now for the beta merge)